force-app/main/default/classes/CredentialExamAttemptService.cls

```diff
@@ -0,0 +1,66 @@
+/**
+ * @author Dipshika Nalawade
+ * @date   04/08/2022
+ * @description Service Class containing methods specific to Credential Exam Attempt
+ */
+public with sharing class CredentialExamAttemptService {
+    /*******************************************************************************************************
+      * @description This method retrieves exam attempts for given user
+      * @param userId SF id of user
+      * @return List of Credential Exam Attempt
+      */
+     @AuraEnabled(cacheable=true)
+     public static List<Credential_Exam_Attempt__c> getExamAttemptsForUser(String userId) {
```

Comment:
remove static, have a separate controller for UI cmp and call service method there.